### ISSUE TITLE
CWR doesn't work with Juju 2.0

### ISSUE THREAD

**Issue by arosales** [juju-solutions/cloud-weather-report#15]
The Readme says to use "controller" as a positional argument to CWR which would imply Juju 2.0, but I was unable to get CWR running against Juju 2.0:

```
arosales@x230:~/devel/cloud-weather-report/build-cloud/test_plans$ juju bootstrap cwr-aws aws
Creating Juju controller "local.cwr-aws" on aws/us-east-1
Bootstrapping model "admin"
Starting new instance for initial controller
Launching instance
i-20de56a7
Installing Juju agent on bootstrap instance
Preparing for Juju GUI 2.1.2 release installation
Waiting for address
Attempting to connect to 54.242.191.215:22
Attempting to connect to 10.31.91.126:22
Logging to /var/log/cloud-init-output.log on remote host
Running apt-get update
Running apt-get upgrade
Installing package: curl
Installing package: cpu-checker
Installing package: bridge-utils
Installing package: cloud-utils
Installing package: cloud-image-utils
Installing package: tmux
Fetching tools: curl -sSfw 'tools from %{url_effective} downloaded: HTTP %{http_code}; time %{time_total}s; size %{size_download} bytes; speed %{speed_download} bytes/s ' --retry 10 -o $bin/tools.tar.gz <[https://juju-dist.s3.amazonaws.com/tools/agent/2.0-beta5/juju-2.0-beta5-xenial-amd64.tgz]>
Bootstrapping Juju machine agent
Starting Juju machine agent (jujud-machine-0)
Bootstrap agent installed
Waiting for API to become available: upgrade in progress (upgrade in progress)
Waiting for API to become available: upgrade in progress (upgrade in progress)
Bootstrap complete, local.cwr-aws now available.
arosales@x230:~/devel/cloud-weather-report/build-cloud/test_plans$ echo $?
0

arosales@x230:~/devel/cloud-weather-report/build-cloud/test_plans$ juju list-controllers
CONTROLLER      MODEL    USER         SERVER
local.cwr-aws*  default  admin@local  54.242.191.215:17070

arosales@x230:~/devel/cloud-weather-report/build-cloud/test_plans$ cwr -v cwr-aws ./realtime-syslog-analytics.yaml 
2016-04-26 13:49:09 INFO Cloud Weather Report started.
Traceback (most recent call last):
  File "/usr/local/bin/cwr", line 9, in <module>
    load_entry_point('cloud-weather-report==0.2.1', 'console_scripts', 'cwr')()
  File "/usr/local/lib/python2.7/dist-packages/cloudweatherreport/run.py", line 21, in entry
    html_filename = main(args)
  File "/usr/local/lib/python2.7/dist-packages/cloudweatherreport/cloud_weather_report.py", line 140, in main
    env = jujuclient.Environment.connect(env_name=env_name)
  File "/usr/lib/python2.7/dist-packages/jujuclient.py", line 534, in connect
    return Connector().run(cls, env_name)
  File "/usr/lib/python2.7/dist-packages/jujuclient.py", line 132, in run
    jhome, data = self.parse_env(env_name)
  File "/usr/lib/python2.7/dist-packages/jujuclient.py", line 224, in parse_env
    raise EnvironmentNotBootstrapped(env_name)
jujuclient.EnvironmentNotBootstrapped: Environment cwr-aws is not bootstrapped

arosales@x230:~/devel/cloud-weather-report/build-cloud/test_plans$ cwr -v aws ./realtime-syslog-analytics.yaml 
2016-04-26 13:48:08 INFO Cloud Weather Report started.
Traceback (most recent call last):
  File "/usr/local/bin/cwr", line 9, in <module>
    load_entry_point('cloud-weather-report==0.2.1', 'console_scripts', 'cwr')()
  File "/usr/local/lib/python2.7/dist-packages/cloudweatherreport/run.py", line 21, in entry
    html_filename = main(args)
  File "/usr/local/lib/python2.7/dist-packages/cloudweatherreport/cloud_weather_report.py", line 140, in main
    env = jujuclient.Environment.connect(env_name=env_name)
  File "/usr/lib/python2.7/dist-packages/jujuclient.py", line 534, in connect
    return Connector().run(cls, env_name)
  File "/usr/lib/python2.7/dist-packages/jujuclient.py", line 132, in run
    jhome, data = self.parse_env(env_name)
  File "/usr/lib/python2.7/dist-packages/jujuclient.py", line 224, in parse_env
    raise EnvironmentNotBootstrapped(env_name)
jujuclient.EnvironmentNotBootstrapped: Environment aws is not bootstrapped



arosales@x230:~$ juju version
2.0-beta5-xenial-amd64
```

**Comment by seman** [comment]
Currently, CWR supports Juju 1.x only. There is a future plan to support Juju 2.

**Comment by arosales** [comment]
`Currently, CWR supports Juju 1.x only. There is a future plan to support Juju 2.
`

Understood, lets keep this bug open until 2.x is supported.

**Comment by secroft** [comment]
What is the status of Juju 2.0 support?

**Comment by johnsca** [comment]
The 2.0 support was fixed in #61 but it seems that master has diverged somewhat since that was submitted and there may be some other 2.0 related changes in master now as well.

**Comment by seman** [comment]
Now it is supported by #61
